sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -0,0 +1,153 @@
+## API Report File for "@azure/web-pubsub"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { HttpRequestBody } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+
+export { AzureKeyCredential }
+
+// @public
+export interface CloseConnectionOptions extends OperationOptions {
+    reason?: string;
+}
+
+// @public
+export interface GetAuthenticationTokenOptions {
+    claims?: {
+        [key: string]: string[];
+    };
+    userId?: string;
+}
+
+// @public
+export interface GetAuthenticationTokenResponse {
+    baseUrl: string;
+    token: string;
+    url: string;
+}
+
+// @public
+export interface GroupAddConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupAddUserOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupAdminClientOptions extends PipelineOptions {
+}
+
+// @public
+export interface GroupHasUserOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupRemoveConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupRemoveUserOptions extends OperationOptions {
+}
+
+// @public
+export interface HasConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface HubAdminClientOptions extends PipelineOptions {
+}
+
+// @public
+export interface HubHasGroupOptions extends OperationOptions {
+}
+
+// @public
+export interface HubHasUserOptions extends OperationOptions {
+}
+
+// @public
+export interface HubRemoveUserFromAllGroupsOptions extends CloseConnectionOptions {
+}
+
+// @public
+export interface HubSendToAllOptions extends OperationOptions {
+    excludedConnections?: string[];
+}
+
+// @public
+export interface HubSendToConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface HubSendToUserOptions extends OperationOptions {
+}
+
+// @public
+export class WebPubSubGroup {
+    // @internal
+    constructor(hubName: string, groupName: string);
```

Comment:
if the constructor is internal, why not use an interface instead? No reason to export the class itself